ObjectOrientedPractics/OOP/View/Tabs/AddressControl.cs

```diff
@@ -0,0 +1,199 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Data;
+using System.Drawing;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows.Forms;
+
+namespace OOP.View
+{
+    public partial class AddressControl : UserControl
+    {
+        /// <summary>
+        /// Копия формы для передачи данных
+        /// </summary>
+        public CustomerTab _customerTab;
+
+        /// <summary>
+        /// Хранит преедан пользователь или нет.
+        /// </summary>
+        private bool _currentCustomer = false;
```

Comment:
Обращение к закрытым полям класса -- зло, даже если внутри. Я уже говорил. Замени на авто свойство.
```suggestion
        /// <summary>
        /// Копия формы для передачи данных
        /// </summary>
        public CustomerTab CustomerTab { get; set; };

        /// <summary>
        /// Хранит преедан пользователь или нет.
        /// </summary>
        private bool CurrentCustomer { get; set; } = false;
```